resources/flavors.json

```diff
@@ -0,0 +1,114 @@
+[
+    {
+      "is_automated": true,
+      "name": "bgruening/galaxy-stable",
+      "is_trusted": true,
+      "is_official": false,
+      "star_count": 8,
```

Comment:
Is it possible to get such information on the fly from the DockerHub API or do we need to update this list?